data/virt_autotest/kubevirt_tests/full-tests.conf

```diff
@@ -35,7 +35,7 @@ GINKGO_FOCUS=container_disk_test.go
 
 [datavolume_test]
 GINKGO_FOCUS=datavolume.go
-GINKGO_SKIP=test_id:3189|test_id:6686|test_id:5252
+GINKGO_SKIP=test_id:3189|test_id:6686|test_id:5252|with Block PVC|with Filesystem PVC|should resolve DataVolume sourceRef|test_id:8571
```

Comment:
Hi Nan, would you please give more details what are skipped and why? I seem to me block and file system PVC are two important types, but now fully skipped?